tagstore/tagstore_rest/v1/views/entities.py

```diff
@@ -62,5 +67,69 @@ def tags(self, request, entity_type=None, pk=None):
             queryset = entity.tags.all()
 
         extracted = [self._convert(tag) for tag in queryset]
-        serializer = EntityTagSerializer({'tags': extracted})
+        return EntityTagSerializer({'tags': extracted})
+
+    def tags(self, request, entity_type=None, pk=None):
+        '''
+        Get a list of all tags belonging to an entity.
+
+        These can be optionally filtered by `taxonomy` params.
+        '''
+        entity = get_object_or_404(Entity, external_id=pk, entity_type=entity_type)
+        taxonomies = request.GET.get('taxonomies', '').split(',')
+
+        serializer = self.serialize_tags(entity, taxonomies)
         return Response(serializer.data)
+
+    def update_tags(self, request, entity_type=None, pk=None):
+        '''
+        Update tags belonging to an entity.
+        '''
+        tagstore = DjangoTagstore()
+
+        entity_obj = get_object_or_404(Entity, external_id=pk, entity_type=entity_type)
+        entity = EntityId(external_id=pk, entity_type=entity_type)
+        freeform_obj, _ = Taxonomy.objects.get_or_create(name='FreeForm')
```

Comment:
I was originally thinking that each user, or perhaps each "course"/"context" on the platform, or perhaps each organization would get its own freeform tag taxonomy, and that such things would be managed by the client (i.e. the LMS or LabXchange), not by Tagstore itself.

However, this approach of having a single FreeForm taxonomy as a first-class thing inside Tagstore makes a lot of sense and is going to help keep things simple at first. I would just ask a few things:
* `get_or_create(name='FreeForm')` will cause issues if someone ever creates another Taxonomy called "FreeForm", which could happen. We shouldn't rely on names being unique. Can we perhaps make a constant like `FREEFORM_TAXONOMY_UID = -100` and load the FreeForm taxonomy by UID? (Using a negative UID guarantees it will never conflict with manually created ones; we could also use the ID `0`).